src/Differ.php

```diff
@@ -97,7 +110,9 @@ public function diff($from, $to, LongestCommonSubsequence $lcs = null)
             }
 
             if ($newChunk) {
-                $buffer  .= "@@ @@\n";
+                if ($this->showNonDiffLines === true) {
+                    $buffer  .= "@@ @@\n";
```

Comment:
As the diff shows, the extra whitespace was there before my changes, but I have updated the diff to resolve this issue accordingly.